telegram_packs/other.txt

```diff
@@ -0,0 +1,88 @@
+other/biFrog|🏳️
+other/croakaColaFrogCan|🧃
+other/doomFrog|🔫
+other/dumbFrog|🤪
+other/eatenFrog|🪦
+other/fingeringFrog|👉
+other/frog2b|🤖
+other/frogBag|💰
+other/frogBatsignal|🌃
+other/frogBeerCan|🍺
+other/frogBeer|🍻🍺
+other/frogBender|🤖
+other/frogBlanket|🛌
+other/frogBlunt|🚬
+other/frogCatCostume|🐱
+other/frogCat|🐱
+other/frogChampagne|🍾
+other/frogChronoTrigger|🐲
+other/frogCoins|🪙
+other/frogDogCostume|🐶
+other/frogDonut|🍩
+other/frogEmil|💀
+other/frogFamilyTkg|👪
+other/frogFlag|🏳️
+other/frogFlash|⚡
+other/frogGun|🔫
+other/frogHacker|🖥️
+other/frogHipster|🤓
+other/frogJester|🃏
+other/frogKaiser|👑
+other/frogKing|👑
+other/frogKirby|🍅
+other/frogMerkel|#️⃣
+other/frogMiddleFinger|🖕
+other/frogMissingTexture|📓
+other/frogMugHand|☕
+other/frogMug|☕
+other/frogMuscleRight|💪
+other/frogMusic|🎧
+other/frogOk|👌
+other/frogPalm|🤦
+other/frogPenguin|🐧
+other/frogPhone|📱
+other/frogPirate|🏴‍☠️
+other/frogPoggers|😏
+other/frogPomelo|🍈
+other/frogPopcorn|🍿
+other/frogPowerslave|🔺
+other/frogPyramid|🔺
+other/frogRainbow|🌈
+other/frogSoviet|🇷🇺
+other/frogSpray|🚿
+other/frogs|🐸
+other/frogStalker|🕵️
+other/frogStop|🛑
+other/frogTear|😵
+other/frogTestPattern|📺
+other/frogThumbsDown|👎
+other/frogThumbsUp|👍
+other/frogTkg|😎
+other/frogTurtle|🐢
+other/frogTuxedoFancy|🤵
+other/frogUwu|😚
+other/frogVodka|🥃
+other/frogWhip|🐸
+other/frogWhisky|🥃
+other/frogWine|🍷
+other/italicFrog|🐸
+other/joyfulFrog|🤣😂
+other/metalFrog|🤘
+other/mineFrog|👉👈
+other/nbFrog|🏳️
+other/pinkFrog|🐸
+other/plagfrog|🐸
+other/pointyFrog|☝️
+other/poisonousFrog|🐸
+other/pregnantFrogCat|🫃🫄🤰
+other/pregnantFrogSun|🫃🫄🤰
+other/prideFrog|🌈
```

Comment:
:rainbow_flag:

Reply:
hmm I might have changed that in the wrong folder :frog: 
will change